tests/baselines/reference/strictOptionalProperties1.errors.txt

```diff
@@ -185,8 +197,6 @@ strictOptionalProperties1.ts(211,1): error TS2322: Type 'string | boolean | unde
     const t2: [number, string?, boolean?] = [1, undefined];
           ~~
 !!! error TS2322: Type '[number, undefined]' is not assignable to type '[number, string?, boolean?]'.
-!!! error TS2322:   Type at position 1 in source is not compatible with type at position 1 in target.

```

Comment:
an elaboration got lost here, it's something I will investigate

Reply:
Elaborations only occur the first time the error is generated - once that result is cached, the elaboration isn't copied to followup errors on the same types. In this case, now this comparison has the same types as the comparison with the error on line 10 of the file, so the elaboration is there now.